projects/media-viewer/src/lib/shared/toolbar-button-toggles.ts

```diff
@@ -16,7 +16,6 @@ export class ToolbarButtonToggles {
   public readonly sidebarOpen = new BehaviorSubject(false);
   public readonly searchBarHidden = new BehaviorSubject(true);
   public readonly drawMode = new BehaviorSubject(false);
```

Comment:
I'm a bit confused, shouldn't draw mode die as well?

Reply:
Yes, however i'm not confident that it's all it does, and i'd be refactoring most of the app.
Its on my target list to get rid of, but in order to get the skeleton in place as requested by everyone this morning, its a 'leave it alone' until i can be sure it doesn't break anything else.

Reply:
gone through and killed this to keep things clean, will be corrected in a coming commit

Reply:
have cleaned this up now @linusnorton can you re-check?